app/sidekiq/benefits_intake_status_job.rb

```diff
@@ -71,12 +71,10 @@ def handle_response(response, pending_form_submissions)
 
     response.body['data']&.each do |submission|
       uuid = submission['id']
-      form_submission = pending_form_submissions.find do |submission_from_db|
-        submission_from_db.latest_attempt&.benefits_intake_uuid == uuid
+      form_submission_attempt = pending_form_submission_attempts.find do |submission_attempt_from_db|
+        submission_attempt_from_db&.benefits_intake_uuid == uuid
       end
```

Comment:
Could change this to create a hash of uuid to attempts to avoid looping the entire list each time

Reply:
Great call! 👍